openfecwebapp/constants.py

```diff
@@ -345,74 +345,220 @@
-    ('transfers_from_affiliated_party', {'label': 'Transfers from affiliated committees', 'level': '2'}), #F3X
-    ('transfers_from_affiliated_committee', {'label': 'Transfer from affiliated committees', 'level': '2'}), #F3P
-    ('transfers_from_other_authorized_committee', {'label': 'Transfer from authorized committees', 'level': '2'}), #F3
-    ('all_loans_received', {'label': 'Loans received', 'level': '2'}), #F3X
-    ('loan_repayments_received', {'label': 'Loan repayments received', 'level': '2'}), #F3X
-    ('loans', {'label': 'Total loans received', 'level': '2'}), # F3
-    ('loans_received', {'label': 'Total loans received', 'level': '2'}), #F3P
-    ('loans_received_from_candidate', {'label': 'Loans made by candidate', 'level': '3'}), #F3P
-    ('loans_made_by_candidate', {'label': 'Loans made by candidate', 'level': '3'}), #F3
-    ('other_loans_received', {'label': 'Other loans', 'level': '3'}), #F3P
-    ('all_other_loans', {'label': 'Other loans', 'level': '3'}), #F3
-    ('total_offsets_to_operating_expenditures', {'label': 'Total offsets', 'level': '2'}), #F3P
-    ('subtotal_offsets_to_operating_expenditures', {'label': 'Offsets to operating expenditures', 'level': '3'}), #F3P
-    ('offsets_to_operating_expenditures', {'label': 'Offsets to operating expenditures', 'level': '2'}), #F3, F3X
-    ('offsets_to_fundraising_expenditures', {'label': 'Fundraising offsets', 'level': '3'}), #F3P
-    ('offsets_to_legal_accounting', {'label': 'Legal and accounting offsets', 'level': '3'}), #F3P
-    ('other_receipts', {'label': 'Other receipts', 'level': '2'}), #F3, F3P
-    ('fed_candidate_contribution_refunds', {'label': 'Candidate refunds', 'level': '2'}), #F3X
-    ('other_fed_receipts', {'label': 'Other Receipts', 'level': '2'}), #F3X
-    ('total_transfers', {'label': 'Total transfers', 'level': '2'}), #F3X
-    ('transfers_from_nonfed_account', {'label': 'Non-federal transfers', 'level': '3'}), #F3X
-    ('transfers_from_nonfed_levin', {'label': 'Levin funds', 'level': '3'}), #F3X
-    ('fed_receipts', {'label': 'Total federal receipts', 'level': '2'}), #F3X
+    ('receipts',  # F3, F3P, F3X
+        {'label': 'Total receipts', 'level': '1', 'term': 'total receipts'}),
+    ('contributions',  # F3, F3P, F3X
+        {'label': 'Total contributions', 'level': '2'}),
+    ('individual_contributions',  # F3, F3P, F3X
+        {'label': 'Total individual contributions', 'level': '3'}),
+    ('individual_itemized_contributions',  # F3, F3P, F3X
+        {'label': 'Itemized individual contributions', 'level': '4',
+            'link': 'individual_contributions'}),
+    ('individual_unitemized_contributions',  # F3, F3P, F3X
+        {'label': 'Unitemized individual contributions', 'level': '4'}),
+    ('political_party_committee_contributions',
+        {'label': 'Party committee contributions', 'level': '3', 'type': {
+            'link': 'receipts',
+                  'P': 'F3P-17B', 'H': 'F3-11B', 'S': 'F3-11B', 'O': 'F3X-11B'
```

Comment:
Just a suggestion for future clarity: what about instead of P, H, S and 0, it was just the form number: 3, 3P, 3X (or 3, P, X)? H and S will always be the same (form 3), so it would also remove an unnecessary item.